gateway/1.10/modules/ROOT/pages/policies-included-xml-threat-protection.adoc

```diff
@@ -0,0 +1,102 @@
+= XML Threat Protection Policy
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images
+:keywords: xml threat protection, api gateway, flex gateway, gateway, policy 
+
+[width="100%", cols="1,3"]
+|===
+>s|Policy Name| XML Threat Protection
+>s|Summary| Protects against malicious XML in API requests
+>s|Category| Security
+>s|First Flex Gateway version available |v1.10.0
+>s|Returned Status Codes|400 - Bad Request
+|===
+
+== Summary
+
+Applications processing XML requests are susceptible to attacks characterized by unusual inflation of elements, attributes, and deep nesting levels. Attackers use recursive techniques to consume memory resources. Dramatic increases in the size of the application data often signal a security risk. Use the XML Threat Protection policy to reject unusual requests that exceed the defined expected size.
+
+== Configuring Policy Parameters
+
+include::partial$policy-title-headers.adoc[tag=configFile]
+
+----
+- policyRef:
+    name: xml-threat-protection
+  config:
+    maxNodeDepth: <number> // OPTIONAL, default: -1
+    maxAttributeCountPerElement: <number> // OPTIONAL, default: -1
+    maxChildCount: <number> // OPTIONAL, default: -1
+    maxTextLength: <number> // OPTIONAL, default: -1
+    maxAttributeLength: <number> // OPTIONAL, default: -1
+    maxCommentLength: <number> // OPTIONAL, default: -1
+----
+
+NOTE: Defining a parameter as `-1` means the parameter has no limit.
+
+[%header%autowidth.spread,cols="a,a,a,a"]
+|===
+|Parameter | Required or Optional | Default Value | Description
```

Comment:
Typically, we use "Required" in the column heading and then ["Yes" or "No"](https://confluence.internal.salesforce.com/spaces/DOCTEAM/pages/143000889/tables+Style+Guidelines#:~:text=because%20of%20localization)-,Yes%20%26%20No,-Supported%20%26%20Not%20Supported) in the row.